environment/ecs_launch_type/ecs_launch_type.go

```diff
@@ -0,0 +1,22 @@
+package ecs_launch_type
```

Comment:
Nit - I don't think packages tend to include `_` or `-`. Kubernetes as an example https://github.com/kubernetes/kubernetes/tree/master/pkg

Reply:
hmm gotcha

Reply:
[TODO]